resources/views/partials/filters/date.blade.php

```diff
@@ -1,4 +1,4 @@
-<div class="dropdown d-inline-block">
+<div class="dropdown d-inline-block" data-controller="screen--filter">
```

Comment:
We need to specify and use this controller in other templates that relate to the filter.